testing/wasm.go

```diff
@@ -1,64 +1,63 @@
 package ibctesting
```

Comment:
I think we can remove this file, the testing pkg should be designed in a way where light clients don't need to put their testing structures here

Reply:
+1, we can delete